src/Microsoft.AspNet.Mvc.Core/Rendering/Html/DefaultHtmlGenerator.cs

```diff
@@ -25,7 +26,7 @@ public class DefaultHtmlGenerator : IHtmlGenerator
             typeof(DefaultHtmlGenerator).GetTypeInfo().GetDeclaredMethod(nameof(ConvertEnumFromString));
 
         private readonly AntiForgery _antiForgery;
-        private readonly IScopedInstance<ActionBindingContext> _bindingContextAccessor;
+        private readonly IClientModelValidatorProvider _clientModelValidatorProvider;
```

Comment:
Part of the work of removing the `OptionDescriptor<T>` is to remove all of the facades that abstract away accessing options, and for our code to just use options directly. What you're doing here isn't exactly that, so it's probably OK if it stays.